src/redux/books/books.js

```diff
@@ -0,0 +1,27 @@
+const ADD_BOOK = 'bookstore/books/ADD_BOOK';
+const REMOVE_BOOK = 'bookstore/books/REMOVE_BOOK';
+
+export const addBook = (book) => ({
+  type: ADD_BOOK,
+  id: book.id,
```

Comment:
- [ ] I don't think we are saving only the book id. We need to store the entire book as well - so there's no need to extract only the id here. We can just have something like this for example (you are free to call `payload` anything you wish or even leave it as `book`):

  ```
  type: ADD_BOOK,
  payload: book
  ```

  Then in the reducer, you need to change `[...state, action.id]` to `[...state, action.payload]`